db/migrate/20201019154701_delete_lingering_subscription_contents.rb

```diff
@@ -0,0 +1,7 @@
+class DeleteLingeringSubscriptionContents < ActiveRecord::Migration[6.0]
+  class SubscriptionContent < ApplicationRecord; end
+
+  def up
+    SubscriptionContent.where("created_at < '2020/06/23'").delete_all
```

Comment:
This date looks a bit dubious - is it tested? I'd expect 2020-06-23 whereas the slashes look like something in between dd/mm/yyyy and yyyy-mm-dd. I'd suggest going dashed even if it works for clarity.

Reply:
Yes it's tested but okay sure